scripts/create_newcase

```diff
@@ -82,10 +82,14 @@ def parse_command_line(args, cimeroot):
                         "with the new user grid changes added to it"
 			"This argument is required if --user-grid is True")
 
-    parser.add_argument("--srcroot",default=os.path.dirname(cimeroot),
+    parser.add_argument("--srcroot", default=os.path.dirname(cimeroot),
                         help="Alternative path for source root directory. By default this is set to"
                         "cimeroot/../")
 
+    if cime_model == "acme":
```

Comment:
If we have time, refactor it.  Otherwise, ask the developer to reimplement with some very clear instructions.